src/olympia/addons/models.py

```diff
@@ -91,6 +92,7 @@
 MAX_SLUG_INCREMENT = 999
 SLUG_INCREMENT_SUFFIXES = set(range(1, MAX_SLUG_INCREMENT + 1))
 GUID_REUSE_FORMAT = 'guid-reused-by-pk-{}'
+UPCOMING_DUE_DATE_CUT_OFF_DAYS_CONFIG_KEY = 'upcoming-due-date-cut-off-days'
```

Comment:
Do we have to have a config ? It adds a SQL query...

Reply:
That was the request from @wagnerand as far I understood it.

Reply:
Yes please. 2 days is too much at the moment, and this allows us to be flexible without a code change and deploy.

Reply:
We used to have caching for the `get_config` stuff but it was often inconsistent and not that useful so we removed it. I don't think it's particularly worth doing for single-key lookups like that. It's fine we can live with the extra (short) query.